ui.frontend/tsconfig.json

```diff
@@ -1,14 +1,27 @@
 {
-  "extends": "@valtech-ch/eslint-config/tsconfig",
   "compilerOptions": {
     "target": "es5",
+    "lib": ["DOM", "DOM.Iterable", "ESNext"],
     "module": "es6",
     "baseUrl": "../ui.frontend",
     "removeComments": true,
     "allowJs": true,
     "preserveConstEnums": true,
     "allowSyntheticDefaultImports": true,
-    "sourceMap": true
+    "esModuleInterop": true,
+    "isolatedModules": true,
+    "sourceMap": true,
+    "moduleResolution": "node",
+    "noEmit": true,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "strictNullChecks": true,
+    "strictPropertyInitialization": true
   },
+  "exclude": ["node_modules", "./babel.config.npm.js"],
```

Comment:
**Thought**

You don't need that babel.config.npm.js in as it is deleted, right?

Reply:
it's used implicitly by babel, that's why it's there (same with many unused dependencies and webpack); I excluded from eslint and tsconfig to avoid an error that would otherwise be thrown

Reply:
ah I saw that you deleted the actual file, then yes, that entry could be removed.